app/services/waste_exemptions_engine/concerns/can_process_jwt_tokens.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require "jwt"
+
+module WasteExemptionsEngine
+  module CanProcessJwtTokens
+    extend ActiveSupport::Concern
+
+    included do
+      def generate_token(payload)
+        JWT.encode(payload, signature, algorithm)
+      end
+
+      def decode_token(token)
+        JWT.decode(token, signature, true, algorithm: algorithm)
+      end
+
+      def signature
+        # TODO: Decide on a more secure algorithm and fix me
```

Comment:
Think we can remove the TODO's now we have our response from security

Reply:
```suggestion
```